app/views.py

```diff
@@ -87,7 +87,7 @@ def files(request, file_):
 @require_POST
 @csrf_exempt
 def deploy(request):
-    forwarded_for = u"{}".format(request.META.get("HTTP_X_FORWARDED_FOR"))
+    forwarded_for = "{}".format(request.META.get("HTTP_X_FORWARDED_FOR"))
```

Comment:
Why's that?

Reply:
I don't know. I haven't changed that. I just used black and I got that. Maybe it was a previous error.

Reply:
Okay, upon checking. This is legacy. In python 2.x, you have to put u before a string to indicate it is Unicode, but in python 3.x this is no longer necessary